### PR TITLE
Update webrender

### PR DESCRIPTION
**Pull request by Manishearth**
Fixes https://github.com/servo/servo/issues/22232 , once https://github.com/servo/webrender/pull/3327 lands and I update this PR to include it

All the integer casts are due to https://github.com/servo/webrender/pull/3291

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/22237)

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -4913,6 +4940,7 @@ dependencies = [
 "checksum ident_case 1.0.0 (registry+https://github.com/rust-lang/crates.io-index)" = "3c9826188e666f2ed92071d2dadef6edc430b11b158b5b2b3f4babbcc891eaaa"
 "checksum idna 0.1.4 (registry+https://github.com/rust-lang/crates.io-index)" = "014b298351066f1512874135335d62a789ffe78a9974f94b43ed5621951eaf7d"
 "checksum image 0.19.0 (registry+https://github.com/rust-lang/crates.io-index)" = "ebdff791af04e30089bde8ad2a632b86af433b40c04db8d70ad4b21487db7a6a"
+"checksum image 0.20.1 (registry+https://github.com/rust-lang/crates.io-index)" = "44665b4395d1844c96e7dc8ed5754782a1cdfd9ef458a80bbe45702681450504"
```
Comment (Eijebong) [inline Cargo.lock]: Wait... Didn't I bum image to 0.20 already ?

Comment (Manishearth) [reply Cargo.lock]: I didn't rebase

**Comment by Manishearth** [comment]
heh, this update wasn't even necessary to diagnose that bug because we already had the relevant commits on master.

Either way, might as well land it.

@bors-servo try

File: Cargo.lock (hunk 1 above)
Comment (Eijebong) [reply Cargo.lock]: Ha :+1: 
Thought I was going crazy

**Comment by Manishearth** [comment]
lol I was only building for android so I didn't do the normal build. Will fix.

**Comment by paulrouget** [comment]
I'm ok with that change, but it doesn't appear to fix #22232.

**Comment by paulrouget** [comment]
>  once servo/webrender#3327 lands

oh :)

**Comment by Manishearth** [comment]
Yeah -- I did all the work for the update whilst bisecting anyway, so made this pr for immediate landing.

And then realized that the fix was in an open WR PR, so I'm just waiting for it to land before I finalize this.

**Comment by atouchet** [comment]
https://github.com/servo/webrender/pull/3338 has been merged.

**Comment by Manishearth** [comment]
@bors r=pcwalton

some other APIs were updated

**Comment by Manishearth** [comment]
@bors-servo r=pcwalton

rustfmt

**Comment by Manishearth** [comment]
@bors-servo retry

connectivity issue on tc

**Comment by Manishearth** [comment]
@bors-servo retry

```
  │ softpipe
  │ 3.3 (Core Profile) Mesa 18.3.0-devel
  │ CloneApi message response was dropped while Webrender was still alive: io error: All senders for this socket closed (thread LayoutThread PipelineId { namespace_id: PipelineNamespaceId(2), index: PipelineIndex(26) }, at /home/servo/.cargo/git/checkouts/webrender-c3596abe1cf4f320/323b7ea/webrender_api/src/api.rs:952)
  │ stack backtrace:
  │    0:     0x7f7eef90ae7c - backtrace::backtrace::trace::h34a5004b5e0f1dc7
  │    1:     0x7f7eef90a0a2 - <backtrace::capture::Backtrace as core::default::Default>::default::hae829d87a5db9150
  │    2:     0x7f7eef90a118 - backtrace::capture::Backtrace::new::hcbbf9ea3df342ffc
  │    3:     0x7f7eed16e209 - servo::main::{{closure}}::hdc2e99ec31dd35c6
  │    4:     0x7f7ef05bd773 - std::panicking::rust_panic_with_hook::hd470ff3b3e7a1b18
  │                         at libstd/panicking.rs:480
  │    5:     0x7f7ef05bd2d9 - std::panicking::continue_panic_fmt::hf0cf39ae1e114602
  │                         at libstd/panicking.rs:390
  │    6:     0x7f7ef05bd23d - std::panicking::begin_panic_fmt::hfc5862442bd06bfc
  │                         at libstd/panicking.rs:345
  │    7:     0x7f7eef939140 - webrender_api::api::RenderApiSender::create_api::hf74055eb268d3d36
  │    8:     0x7f7eed51454f - layout_thread::LayoutThread::new::he32ffb7ec737bea9
  │    9:     0x7f7eed5e256c - std::sys_common::backtrace::__rust_begin_short_backtrace::h3cb885684a401e2e
  │   10:     0x7f7eed5e2be5 - _ZN3std9panicking3try7do_call17hbeadb4104282b3acE.llvm.4581322324593936562
  │   11:     0x7f7ef05cf9e9 - __rust_maybe_catch_panic
  │                         at libpanic_unwind/lib.rs:102
  │   12:     0x7f7eed654590 - <F as alloc::boxed::FnBox<A>>::call_box::hf20116e0db68297e
  │   13:     0x7f7ef05ba65a - <alloc::boxed::Box<(dyn alloc::boxed::FnBox<A, Output$u3d$R$GT$$u20$$u2b$$u20$$u27$a$RP$$GT$$u20$as$u20$core..ops..function..FnOnce$LT$A$GT$$GT$::call_once::h9d1cd498499ed135
  │                         at liballoc/boxed.rs:682
  │                          - std::sys_common::thread::start_thread::ha4bfe77455e6e434
  │                         at libstd/sys_common/thread.rs:24
  │   14:     0x7f7ef05a7525 - std::sys::unix::thread::Thread::new::thread_start::hebe133da3ea5230a
  │                         at libstd/sys/unix/thread.rs:90
  │   15:     0x7f7eeadd7183 - start_thread
  │   16:     0x7f7ee969e03c - clone
  │   17:                0x0 - <unknown>
  │ ERROR 2018-11-28T00:28:14Z: servo: CloneApi message response was dropped while Webrender was still alive: io error: All senders for this socket closed
  └ Pipeline failed in hard-fail mode.  Crashing!
```

may be an intermittent?

**Comment by Manishearth** [comment]
Ah, it's the same intermittent as in https://github.com/servo/servo/issues/20633

the test doesn't try to render much so I suspect this is a general intermittent

**Comment by Manishearth** [comment]
@bors-servo r=pcwalton

so many new builders to gate on

**Comment by Manishearth** [comment]
@bors-servo retry

 - https://github.com/servo/servo/issues/21067

**Pull request merged**